client/src/components/notebook/exporters/toSAS.ts

```diff
@@ -36,3 +40,9 @@ submit;
 ${code}
 endsubmit;
 run;`;
+
+const wrapRlang = (code: string) => `proc rlang;
```

Comment:
This is duplicated here and in `SASCodeDocument`, which isn't ideal, but I see the duplication happening for other code types as well. 

I'll make a todo to consolidate all of these calls in a separate PR